### ISSUE TITLE
Going from layer -> OTLP collector -> Jaeger, causes a warning to be displayed in Jaeger

### ISSUE THREAD

**Issue by MidasLamb** [davidB/axum-tracing-opentelemetry#31]
When opening a root trace generated by this library, the following warning is displayed in Jaeger:
```
    invalid parent span IDs=c1b973f219df2f05; skipping clock skew adjustment
```
![image](https://user-images.githubusercontent.com/14802865/199937108-4e2d5c03-7879-4ae5-bebc-22ec2d174e72.png)

This is a standalone server for now, so nothing is sending trace ids (yet), it just receives raw requests from a browser.

**Comment by davidB** [comment]
Currently the layer, automatically generate a trace_id (+span_id fake) if no trace/span is provide by the client (that will become an option). Maybe it's the cause.

Thanks for reporting

**Comment by MidasLamb** [comment]
I've deployed it in a test environment now and honeycomb is also pointing this out: 
![image](https://user-images.githubusercontent.com/14802865/199983971-82ea502d-e52a-45e1-a2de-843326c95cda.png).

Thanks for the quick replies and this crate in general. I just had to set it up for work today so I made the issues with things I came across when setting it up for the first time and I didn't have much time to really dig deep in them, so apologize if they were misconstrued as being annoyed or if I came across unhappy with this crate, I just quickly made the issues before I forgot about them.
That being said, awesome crate (especially the little touch of the function to get the current trace id), and I'd like to see this crate gain even more traction! So if you're OK with it I can potentially look into making a PR for this (probably somewhere next week).
